frontend/src/components/admin/ManageStories.tsx

```diff
@@ -1,4 +1,5 @@
 import React, { useState } from "react";
+import { useHistory } from "react-router-dom";
```

Comment:
can you also make the import button functional on the ManageStoryTranslations page?